mycroft_admin_service.py

```diff
@@ -140,19 +142,48 @@ def system_reboot(*_):
     call('systemctl reboot -i', shell=True)
 
 
-def system_update(client, data):
+def update_only_mycroft():
+    call(['apt-get', 'update', '-o', 'Dir::Etc::sourcelist="sources.list.d/repo.mycroft.ai.list"',
+          '-o', 'Dir::Etc::sourceparts="-"', '-o', 'APT::Get::List-Cleanup="0"'])
+
+
+def get_core_version():
+    lines = check_output(['dpkg', '--list']).encode().split('\n')
```

Comment:
Should this be `decode()` instead of encode()? On python3 the result of check_output is bytes and to turn that into a string decode() would be used...

Reply:
Nice catch! I should've realized that. Updated.